extensions/amp-analytics/0.1/vendors/googleanalytics.json

```diff
@@ -2,7 +2,7 @@
   "vars": {
     "eventValue": "0",
     "documentLocation": "SOURCE_URL",
-    "clientId": "CLIENT_ID(AMP_ECID_GOOGLE,,_ga)",
+    "clientId": "CLIENT_ID(AMP_ECID_GOOGLE,,_ga,true)",
```

Comment:
do you also need to update https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/test/vendor-requests.json#L157